servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/TerminalSignalConsumer.java

```diff
@@ -40,4 +42,14 @@ default void onError(Throwable throwable) {
      */
     default void onCancel() {
     }
+
+    /**
+     * Creates a new {@link TerminalSignalConsumer} from {@link Runnable}.
+     *
+     * @param onFinally a {@link Runnable} to run on any terminal signal
+     * @return a new {@link TerminalSignalConsumer} from {@link Runnable}
+     */
+    static TerminalSignalConsumer from(Runnable onFinally) {
+        return new RunnableTerminalSignalConsumer(onFinally);
```

Comment:
Tried to place `RunnableTerminalSignalConsumer` in some internal module to avoid adding this API, but the module structure did not allow me to reuse this class in `concurrent-api` and `http-utils`.

Reply:
It seems Ok to me to add as a `public` method.